### PR TITLE
feat: make the cards clickable

### LINKED ISSUE #17: feat: make the card clickable (not the blue button)
To Do:
- Make the card clickable
- Make the button unclickable

image

### PR DESCRIPTION
**Pull request by Panquesito7**
## Related Issue 

Closes #17

### Describe the changes you've made

- Make the cards clickable.
  - In the future, we can make those cards redirect to another site.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

I don't think this applies here.

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -161,6 +162,7 @@ <h1 class="w-full my-2 text-4xl font-bold leading-tight text-center text-gray-80
           <div class="h-1 mx-auto gradient w-64 opacity-25 my-0 py-0 rounded-t"></div>
         </div>
 
+        <div class="card">
```
Comment (mkubdev) [inline index.html:165]: What do you think about [Pointer Events](https://tailwindcss.com/docs/pointer-events) ? I think we should keep the Tailwind's class.

**Review by mkubdev — COMMENTED** [review]
LGTM but i think we can improve it with [Pointer Events](https://tailwindcss.com/docs/pointer-events)

File: index.html (hunk 1 above)
Comment (mkubdev) [reply index.html:165]: Like 'cursor-pointer' class

Comment (vinzvinci) [reply index.html:165]: Yeah, I think we should keep the Tailwind

Comment (Panquesito7) [reply index.html:165]: Sounds good! Should we use the `cursor-pointer` class, then?

**Review by shuklaritvik06 — APPROVED** [review]
CITY :) Let's wait for mkubdev review :)

**Review by mkubdev — APPROVED** [review]
LGTM 🔥

**Pull request merged**